docs/sdk/api/syncActions.md

```diff
@@ -556,3 +558,37 @@ client.execute(statesRequests)
 .then(result => ...)
 .catch(error => ...)
 ```
+
+---
+
+### Note
+
+When it comes to value evaluation for fields of possibly any type, is that we won't generate any `update-actions` for you in these following cases.
+
+| Before      | Now         | Will generate update action? | Value for the action    |
+| ----------- | ----------- | ---------------------------- | ----------------------- |
+| `""`        | `null`      | no                           |                         |
```

Comment:
After discussing with @butenkor, it looks like this is supposed to generate an update action,
where we omit where the value is omitted from the action. good catch 🎣